packages/web-react/src/components/TextFieldBase/README.md

```diff
@@ -3,24 +3,23 @@
+<TextFieldBase id="example" label="Example multiline TextFieldBase" name="example" isMultiline isRequired validationState="error" message="validation failed" />
 ```
 
 ## Available props
 
-| Name              | Type                          | Description                       |
-| ----------------- | ----------------------------- | --------------------------------- |
-| `id`              | string                        | Input and label identification    |
-| `name`            | string                        | Input name                        |
-| `type`            | `text`, `password`, `email`   | Input type                        |
-| `label`           | string                        | Label text                        |
-| `placeholder`     | string                        | Input placeholder                 |
-| `value`           | string                        | Input value                       |
-| `message`         | string                        | Validation or help message        |
-| `validationState` | `success`, `warning`, `error` | Type of validation state          |
-| `isDisabled`      | boolean                       | Whether is field disabled         |
-| `isMultiline`     | boolean                       | Whether is DOM element `textarea` |
-| `isRequired`      | boolean                       | Whether is field required         |
-| `isLabelHidden`   | boolean                       | Whether is label hidden           |
+| Name              | Type                                                          | Description                       |
+| ----------------- | ------------------------------------------------------------- | --------------------------------- |
+| `id`              | string                                                        | Input and label identification    |
+| `isDisabled`      | boolean                                                       | Whether is field disabled         |
+| `isLabelHidden`   | boolean                                                       | Whether is label hidden           |
+| `isMultiline`     | boolean                                                       | Whether is DOM element `textarea` |
+| `isRequired`      | boolean                                                       | Whether is field required         |
+| `label`           | string                                                        | Label text                        |
+| `message`         | string                                                        | Validation or help message        |
+| `name`            | string                                                        | Input name                        |
+| `placeholder`     | string                                                        | Input placeholder                 |
+| `type`            | `email`, `number`, `password`, `search`, `tel`, `text`, `url` | Input type                        |
```

Comment:
This is also in the `web-twig` package. Can you fix it there as well? Or is it gonna be another PR?

Reply:
My bad, just saw it. Sorry.